### PR TITLE
Feat/214/setup hero donate page

### PR DESCRIPTION
**Pull request by littilemike11**
## What changed?
added hero section to new donate pg, added new img from figma to /images, created herosection component for donate

#214 
## How will this change be visible?
go to path /donate/new-donate on website, I was unsure how to change internal webpage links, meaningt he donate button still goes to the previosu version

## How can you test this change?

- [ ] Automated tests
- [x] Manual tests (describe)
determine if hero section matches figma design, also need to double check what the the links for the donate buttons

### CODE REVIEW

File: src/components/donate/HeroSection.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+              <Flex
+                direction={"column"}
+                position={"absolute"}
+                bottom={"0"}
+                left={{ sm: "-27%" }}
+                gap={"7"}
+                px={"4"}
+                width={"100%"}
+              >
+                <Box width={"600px"}>
```
Comment (jtfairbank) [inline src/components/donate/HeroSection.tsx:40]: **suggestion (blocking):** The static width is creating layout issues on smaller screen sizes (less than 600px).  Can we use Radix's breakpoints to just making it 100% for the `initial` breakpoint (0px or larger), and set a min / max width at the `xs` breakpoint (520px or larger)?

File: src/components/donate/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                  <Text size={{ initial: "8", sm: "9" }} className="text-white">
```
Comment (jtfairbank) [inline src/components/donate/HeroSection.tsx:41]: **nitpick:** As the page title, this should probably be a `Heading` component with the `as` attribute set to `h1`.

File: src/components/donate/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                <Flex
+                  p={"7"}
+                  direction={"column"}
+                  style={{ background: "#DFCDE8" }}
+                  width={{ initial: "100%", sm: "670px" }}
+                  height={{ initial: "100%", sm: "180px" }}
+                  gap={{ initial: "3", sm: "5" }}
+                  className="z-10 rounded-bl-[32px]"
+                >
+                  <Text
```
Comment (jtfairbank) [inline src/components/donate/HeroSection.tsx:55]: **nitpick:** As a section title, this should probably be a `Heading` component with the `as` attribute set to `h2`.

**Review by jtfairbank — CHANGES_REQUESTED** [review]
**Praise:** @littilemike11 great job on a fairly complex layout!  Got a few suggestions for you.

One thing to consider is re-approaching this to minimize the amount of content that is `absolute` positioned.  Based on the design, it seems like you could divide the content into 3 sections:

1. The "support us today" content in a relatively positioned wrapper with a blue background and top padding.
2. The "ship aid to people in need" content in a relatively positioned wrapper with a white background (the content itself would have the purple background).
3. The hero image, which could be absolutely positioned.  Though you might be able to get away with including it in #2 and using relative positioning to move it up.

I'm not saying to redo your whole approach right now, especially since we're trying to get this out the door!  But it might be a fun thing to try to refactor in an iteration?

**Comment by littilemike11** [comment]
addressed the specific nitpicks and suggestion (blocking) comments for, but may need more time to think of a less absolute positioned approach.

**Review by jtfairbank — APPROVED** [review]
**praise:** Thanks for the touchups @littilemike11!

I've made an issue to attempt minimizing the absolute positioning (see #288).

**Pull request merged**